### PR TITLE
[BM-2932] Update Landing Page Text

### PR DESCRIPTION
**Pull request by vicenters1**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Test A
- [x] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: components/sections/DefinitionSection.jsx [hunk 1]
```diff
@@ -14,10 +14,8 @@ export default function DefinitionSection() {
                 </div>
                 <div className="lg:ml-4 mt-5 lg:self-center lg:mt-0 flex flex-col gap-y-3 tracking-tight lg:tracking-normal text-justify text-lg lg:text-xl xl:text-2xl font-semibold text-dark-brown lg:mx-[5%] lg:w-[40%] lg:gap-y-9 drop-shadow-sm">
                     <p>
-                        <i>
-                            Uma <b>Beer Mile</b> é uma prova de carácter desportivo e
-                            recreativo que combina a corrida e a cerveja.
-                        </i>
+                        Uma <b>Beer Mile</b> é uma prova de carácter desportivo e
+                        recreativo que combina a corrida e a cerveja.
```
Comment (vicenters1) [inline components/sections/DefinitionSection.jsx]:
```suggestion
                        Uma <b>Beer Mile</b> é uma prova de carácter desportivo e
                        recreativo que combina a cerveja e a cerveja.
```

**Review by vicenters1 — COMMENTED** [review]
Everything else LGTM

File: components/sections/DefinitionSection.jsx (hunk 1 above)
Comment (jaimemsaramago) [reply components/sections/DefinitionSection.jsx]: boa vicente

**Review by jaimemsaramago — APPROVED** [review]
LGTM

**Pull request merged**